nms/app/packages/magmalte/app/views/equipment/EnodebDetailMain.js

```diff
@@ -18,26 +18,28 @@ import type {WithAlert} from '@fbcnms/ui/components/Alert/withAlert';
 import Button from '@material-ui/core/Button';
 import CardTitleRow from '../../components/layout/CardTitleRow';
 import DashboardIcon from '@material-ui/icons/Dashboard';
+import DataUsageIcon from '@material-ui/icons/DataUsage';
 import DateTimeMetricChart from '../../components/DateTimeMetricChart';
 import EnodebConfig from './EnodebDetailConfig';
 import EnodebContext from '../../components/context/EnodebContext';
 import GatewayLogs from './GatewayLogs';
-import GraphicEqIcon from '@material-ui/icons/GraphicEq';
 import Grid from '@material-ui/core/Grid';
 import React from 'react';
 import SettingsIcon from '@material-ui/icons/Settings';
-import SettingsInputAntennaIcon from '@material-ui/icons/SettingsInputAntenna';
+import Text from '../../theme/design-system/Text';
 import TopBar from '../../components/TopBar';
 import nullthrows from '@fbcnms/util/nullthrows';
 import withAlert from '@fbcnms/ui/components/Alert/withAlert';
 
+import moment from 'moment';
```

Comment:
minor nit: Move the extra line above to be between the 
`import moment from 'moment';` and `import {DateTimePicker} from '@material-ui/pickers';`
Similarly remove additional empty lines above and separate the type imports and rest of the imports with a empty line